packages/migration/src/migrate.ts

```diff
@@ -57,12 +62,12 @@ import changeBaseColorToHex from './migrations/v0036_changeBaseColorToHex';
 import removeTransientMediaProperties from './migrations/v0037_removeTransientMediaProperties';
 import camelCaseResourceSizes from './migrations/v0038_camelCaseResourceSizes';
 import backgroundAudioFormatting from './migrations/v0039_backgroundAudioFormatting';
-import andadaFontToAndadaPro from './migrations/v0040_andadaFontToAndadaPro.js';
-import removeFontProperties from './migrations/v0041_removeFontProperties.js';
-import removeTrackName from './migrations/v0042_removeTrackName.js';
+import andadaFontToAndadaPro from './migrations/v0040_andadaFontToAndadaPro';
+import removeFontProperties from './migrations/v0041_removeFontProperties';
+import removeTrackName from './migrations/v0042_removeTrackName';
 import removeTagNames from './migrations/v0043_removeTagNames';
 
-const MIGRATIONS = {
+const MIGRATIONS: Record<number, any[]> = {
```

Comment:
Still don't have a solution for a generic `MigrationFn` type yet (instead of `any[]`). Any (non-`any` 😛 ) ideas are appreciated.

Reply:
It's a tough one. Unfortunately [the article Morten shared before](https://blog.saleae.com/versioning-typescript-types/) doesn't really cover that for a case like ours.

I've briefly looked into using union types here or something else clever, but couldn't find a nice way for it.

Soo... maybe we just use `any` for now until we find something better? 🙈